### DISCUSSION TITLE
How can I access the system's planet data?

### DISCUSSION THREAD

**Discussion by YenShihHung** [EDCD/EDDI#2675] [EDDI VoiceAttack Q&A]
hello i have a question
I want to report the star space I am in when entering super cruise
For example, entering the second satellite airspace

However, EDDI does not see the list of planets in the VA variable. How can I access the planet data of the system?

**Comment by Tkael** [comment]
<https://github.com/EDCD/EDDI/wiki/Near-surface-event> might work for that use case.
